.env.sample

```
@@ -1,6 +1,29 @@
-# REACT_APP_API_URL=http://localhost:4000
-# NODE_ENV=development
-# MONGO_URL=mongodb://localhost:27017/erxes
-# NODE_OPTIONS=--max-old-space-size=1536
-# ENABLED_PLUGINS=operation
-# DISABLE_CHANGE_STREAM=true
+JWT_TOKEN_SECRET=CHANGE_ME_use_openssl_rand_hex_32
+
+DOMAIN=http://localhost:3001
+REACT_APP_API_URL=http://localhost:4000
+REACT_APP_API_SUBSCRIPTION_URL=ws://localhost:4000/graphql
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The REACT_APP_API_SUBSCRIPTION_URL key should go before the REACT_APP_API_URL key

(UnorderedKey)